src/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Manager.cpp

```diff
@@ -207,17 +195,11 @@ Real Manager::getUt1MinusUtcAt(const Instant& anInstant) const
 
             return prediction.ut1MinusUtc;
         }
-        else
-        {
-            throw ostk::core::error::RuntimeError(
-                "Cannot obtain UT1 - UTC from Bulletin A at [{}].", anInstant.toString()
-            );
-        }
     }
 
-    const Finals2000A* finals2000aPtr = this->accessFinals2000AAt(anInstant);
+    const Finals2000A* finals2000aPtr = this->accessFinals2000A();
 
-    if (finals2000aPtr != nullptr)
+    if (finals2000aPtr)
```

Comment:
```suggestion
    if (finals2000aPtr != nullptr)
```
Nit: I believe this is more explicit and thus preferred.

Reply:
I'm fine with either. Though we had decided on the other though 🤔

Reply:
I was super inconsistent in where I changed this anyway. Reverted!